backends/bmv2/common/expression.h

```diff
@@ -119,6 +119,7 @@ class ExpressionConverter : public Inspector {
     void postorder(const IR::PathExpression* expression) override;
     void postorder(const IR::TypeNameExpression* expression) override;
     void postorder(const IR::Expression* expression) override;
+    void mapExpression(const IR::Expression* expression, Util::IJson* json);
```

Comment:
I suppose you added these for debugging ... is it worth keeping?

Reply:
Yes, in fact is is very useful.